### PR TITLE
Corrected grammar in strings.xml

### PR DESCRIPTION
**Pull request by Atrate**

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -31,73 +31,73 @@
...
+    <string name="gps_processed">GPS request processed!</string>
     <string name="from">from \"%1$s\"</string>
-    <string name="wifi_processed">"Wifi-request processed! "</string>
-    <string name="remote_adding">Remote phone number adding!</string>
+    <string name="wifi_processed">"Wifi request processed! "</string>
+    <string name="remote_adding">Remote phone number added!</string>
     <string name="was_added">%1$s added to trusted</string>
     <string name="menu_gps_send">Send GPS data</string>
-    <string name="menu_wifi_send">Send network data</string>
+    <string name="menu_wifi_send">Send WiFi data</string>
```
Comment (Seva-coder) [inline app/src/main/res/values/strings.xml]: app will send not only wifi mac's, but base stations ID too..

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -31,73 +31,73 @@
...
     <string name="coordinates_will_be_sent">Coordinates (using GPS) will be sent to %1$s</string>
-    <string name="nets_will_be_sent">Network data will be sent to %1$s</string>
-    <string name="no_gps_or_rights">GPS disabled/no permissions! Location will not be sent!</string>
-    <string name="no_rights_for_wifi_sending">Not enough permissions, data is not sent!</string>
-    <string name="permissions_obtained">permissions - OK</string>
-    <string name="no_permits_received">Not enough permissions!</string>
+    <string name="nets_will_be_sent">WiFi data will be sent to %1$s</string>
```
Comment (Seva-coder) [inline app/src/main/res/values/strings.xml]: same as in the previous

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -31,73 +31,73 @@
     <string name="altitude">Altitude: %1$.0f m\n</string>
     <string name="speed">Speed: %1$.2f km/h\n</string>
     <string name="course">Bearing: %1$.0f°\n</string>
-    <string name="delete_warning">Delete a location from history?  (irreversibly!)</string>
+    <string name="delete_warning">Delete a location entry from history? (Irreversible!)</string>
     <string name="yes">Yes</string>
     <string name="no">No</string>
-    <string name="no_internet_warning">Network disconnected, send a request anyway? (without the Internet, the wifi request will have to be reopened from the list of SMS messages when the Internet will be on)</string>
-    <string name="no_rights_for_sms">Not enough permissions, request is not sent!</string>
+    <string name="no_internet_warning">Wifi  disconnected, send a request anyway? (without wifi, the request will have to be reopened from the list of SMS messages after connecting to wifi.)</string>
```
[the remark this answers is not in the document]
Comment (Atrate) [reply app/src/main/res/values/strings.xml]: Oh yes, that makes sense, will fix soon!

**Comment by Seva-coder** [comment]
Looks good,
thanks!

**Pull request merged**